js/filters/genericFilters.js

```diff
@@ -59,5 +59,33 @@
 			var integerPart = text.split(separator)[0];
 			return $sce.trustAsHtml(integerPart + rightSpan);
 		};
-	}]);
-})();
\ No newline at end of file
+	}])
```

Comment:
could you add it to the typescript section of the code, also also add some unit test